disnake/flags.py

```diff
@@ -151,6 +132,26 @@ def __init__(self, **kwargs: bool):
                 raise TypeError(f"{key!r} is not a valid flag name.")
             setattr(self, key, value)
 
+    @classmethod
+    def __init_subclass__(cls, inverted: bool = False, no_fill_flags: bool = False):
+        # add a way to bypass filling flags, eg for ListBaseFlags.
+        if no_fill_flags:
+            return cls
+        cls.VALID_FLAGS = {}
```

Comment:
This clears `VALID_FLAGS` and `DEFAULT_VALUE` in subclasses of flag types, which would be a regression from previous behavior where this only happened in subclasses decorated with `@fill_with_flags`.

Reply:
https://github.com/DisnakeDev/disnake/pull/660/commits/9c893cb5899c334bb0fb15dd0f086092dba384da